frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -56,3 +56,5 @@ export const listPage = {
       cy.get(`[data-test-id="${resourceName}"]`, { timeout: 90000 }).should('not.be.visible'),
   },
 };
+
+export const status = '[data-test="camel-case-wrapped-text"]';
```

Comment:
This isn't very specific to `status`.  There is a potential for a list page to have multiple tables, sections, statuses.  Can we use something more specific, ie `data-test='pvc-status'`.
Also, not sure `status` is common enough on list pages to warrant putting it in `list-page.ts`.